workflows/wf_theiaeuk_pe.wdl

```diff
@@ -0,0 +1,171 @@
+    Int min_reads = 30000
+    #Edited default values
+    Int min_basepairs = 90000000
+    Int min_genome_size = 9000000
+    Int max_genome_size = 178000000
+    Int min_coverage = 10
+    Int min_proportion = 50
+    Boolean skip_screen = false 
+  }
+  call versioning.version_capture{
+    input:
+  }
+  call screen.check_reads as raw_check_reads {
+    input:
+      read1 = read1_raw,
+      read2 = read2_raw,
+      min_reads = min_reads,
+      min_basepairs = min_basepairs,
+      min_genome_size = min_genome_size,
+      max_genome_size = max_genome_size,
+      min_coverage = min_coverage,
+      min_proportion = min_proportion,
+      skip_screen = skip_screen
+  }
+  if (raw_check_reads.read_screen=="PASS") {
+    call read_qc.read_QC_trim {
+      input:
+        samplename = samplename,
+        read1_raw = read1_raw,
+        read2_raw = read2_raw
+    }
+    call screen.check_reads as clean_check_reads {
+      input:
+        read1 = read_QC_trim.read1_clean,
+        read2 = read_QC_trim.read2_clean,
+        min_reads = min_reads,
+        min_basepairs = min_basepairs,
+        min_genome_size = min_genome_size,
+        max_genome_size = max_genome_size,
+        min_coverage = min_coverage,
+        min_proportion = min_proportion,
+        skip_screen = skip_screen
+    }
+    if (clean_check_reads.read_screen=="PASS") {
+      call shovill.shovill_pe {
+        input:
+          samplename = samplename,
+          read1_cleaned = read_QC_trim.read1_clean,
+          read2_cleaned = read_QC_trim.read2_clean
+      }
+      call quast.quast {
+        input:
+          assembly = shovill_pe.assembly_fasta,
+          samplename = samplename
+      }
+      call cg_pipeline.cg_pipeline {
+        input:
+          read1 = read1_raw,
+          read2 = read2_raw,
+          samplename = samplename,
+          genome_length = clean_check_reads.est_genome_length
+      }
+      call gambit.gambit_euk as gambit {
+        input:
+          assembly = shovill_pe.assembly_fasta,
+          samplename = samplename
+      }
+      call ts_mlst.ts_mlst {
+        input: 
+          assembly = shovill_pe.assembly_fasta,
+          samplename = samplename
+      }
+      call merlin_magic.merlin_magic {
+        input:
+          merlin_tag = gambit.merlin_tag,
+          assembly = shovill_pe.assembly_fasta,
+          samplename = samplename,
+          read1 = read_QC_trim.read1_clean,
+          read2 = read_QC_trim.read2_clean
+      }
+    }
+  }
+  output {
+    # Version Captures
+    String theiaeuk_illumina_pe_version = version_capture.phbg_version
+    String theiaeuk_illumina_pe_analysis_date = version_capture.date
+    # Read Metadata
+    String seq_platform = seq_method
+    # Sample Screening
+    String theiaeuk_raw_read_screen = raw_check_reads.read_screen
+    String? theiaeuk_clean_read_screen = clean_check_reads.read_screen
+    # Read QC
+    Int? theiaeuk_num_reads_raw1 = read_QC_trim.fastq_scan_raw1
```

Comment:
is there a reason everything is prefaced with theiaeuk_?

Reply:
Andrew asked this same question, and it was originally made that way to distinguish between output columns generated by theiaprok and theiaeuk run on the same samples. Since that shouldn't happen anymore we can def remove that.